src/sx-design/src/ErrorBoundary/ErrorBoundary.js

```diff
@@ -90,57 +97,33 @@ export default class ErrorBoundary extends React.Component<Props, State> {
 
 const styles = sx.create({
   error: {
-    color: 'rgb(var(--sx-foreground))',
-    background: 'rgb(var(--sx-background))',
-    fontFamily:
-      '-apple-system, BlinkMacSystemFont, Roboto, "Segoe UI", "Fira Sans", Avenir, "Helvetica Neue", "Lucida Grande", sans-serif',
     height: '100vh',
     textAlign: 'center',
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
     justifyContent: 'center',
-  },
-  desc: {
-    display: 'inline-block',
-    textAlign: 'left',
-    lineHeight: '49px',
-    height: 49,
-    verticalAlign: 'middle',
+    gap: '2rem',
   },
   h1: {
-    display: 'inline-block',
     borderInlineEnd: '1px solid rgba(var(--sx-foreground), 0.3)',
-    margin: 0,
-    marginInlineEnd: 20,
-    paddingBlock: '10px',
-    paddingInlineEnd: '23px',
-    paddingInlineStart: '0px',
-    fontSize: 24,
-    fontWeight: 500,
-    verticalAlign: 'top',
```

Comment:
This is one of the motivations: many custom styles can be now removed and it looks better at the end.